include/vrf.h

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Tuposoft Collective on 23.01.2023.
+//
+
+#pragma once
+
+#define SMTP_DATA_LINES_MAX_LENGTH 998
+
+#include <stdbool.h>
```

Comment:
Do we need this included here, I don't see any boolean usage in this file? We should probably move it to the .c file.